tests/fe/nedelec_embed_dofs.cc

```diff
@@ -0,0 +1,90 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 - 2021 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+// Checks if get_embedding_dofs() correctly maps the local DoF indices
+// of an FE_Nedelec element of sub_degree to an element of sup_degree,
+// where sub_degree <= sup_degree
+
+#include <deal.II/base/quadrature_lib.h>
+
+#include <deal.II/fe/fe_nedelec.h>
+
+#include "interpolate_common.h"
+
+template <unsigned int dim>
+void
+check_nedelec_embed(const unsigned int sub_degree,
+                    const unsigned int sup_degree)
+{
+  Assert((sub_degree > 0) && (sub_degree <= sup_degree),
+         ExcIndexRange(sub_degree, 1, sup_degree));
+
+  FE_Nedelec<dim> fe_sub(sub_degree - 1), fe_sup(sup_degree - 1);
+
+  deallog << fe_sub.get_name() << ' ' << fe_sup.get_name() << ' '
+          << fe_sub.dofs_per_cell << ' ' << fe_sup.dofs_per_cell;
+
+  // Set the quadrature sufficiently high for the enriched finite element space
+  QGauss<dim> quadrature(fe_sup.degree + 1);
+
+  std::vector<double> dofs_sub(fe_sub.dofs_per_cell),
+    dofs_sup(fe_sup.dofs_per_cell);
+
+  // Assign arbitrary values to dofs_sub
+  for (unsigned int i = 0; i < dofs_sub.size(); ++i)
+    dofs_sub[i] = sin(i + 0.5);
```

Comment:
```suggestion
    dofs_sub[i] = std::sin(i + 0.5);
```
Only cosmetic change. I'd suggest to be more specific here.